Lib/test/test_lltrace.py

```diff
@@ -0,0 +1,43 @@
+
+import os
+import tempfile
+import textwrap
+import unittest
+
+from test import support
+from test.support.script_helper import (
+    assert_python_ok, assert_python_failure, interpreter_requires_environment,
+)
+
+
+class TestLLTrace(unittest.TestCase):
+
+    def test_lltrace_does_not_crash_on_subscript_operator(self):
+        # If this test fails, it will reproduce a crash reported as
+        # bpo-34113. The crash happened at the command line console of
+        # debug Python builds with __ltrace__ enabled (only possible in console),
+        # when the interal Python stack was negatively adjusted
+        with tempfile.NamedTemporaryFile(mode='w+', delete=False) as tmp_script:
```

Comment:
I don't think that it's a good practice to keep the file open while running the script in a subprocess.

I suggest to use support.TESTFN filename, and use self.addCleanup(support.remove, support.TESTFN) before creating the file.